src/pixelator/analysis/colocalization/__init__.py

```diff
@@ -223,7 +215,9 @@ def colocalization_scores(
     :param min_region_count: The minimum size of the region (e.g. number
                              of counts in the neighbourhood) required
                              for it to be considered
+    :param random_seed: Set a random seed for the permutation function
```

Comment:
Should we add this to the CHANGELOG? Is it important enough?

Reply:
I don't think it's necessary, since it basically only comes into play in testing scenarios.